### PR TITLE
Scan report

### PR DESCRIPTION
**Pull request by shanimal08**
### References Issues:
https://app.asana.com/0/1158142652422330/1166002911064692
### Description:
Scan count report
### Release Screenshots / Video:
image

### Environment Variables:
 * No change

### CODE REVIEW

File: src/components/pages/admin/reports/eventScanCount/EventScanCountRow.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import PropTypes from "prop-types";
+import { withStyles, Typography } from "@material-ui/core";
+import classNames from "classnames";
+import {
+	fontFamilyDemiBold,
+	fontFamily,
+	primaryHex
+} from "../../../../../config/theme";
+
+const styles = theme => {
```
Comment (msbodetti) [inline src/components/pages/admin/reports/eventScanCount/EventScanCountRow.js]: Let's move styles to the end of the file :)

Comment (shanimal08) [reply src/components/pages/admin/reports/eventScanCount/EventScanCountRow.js]: forgot about that! thanks :)

File: src/components/pages/admin/reports/eventScanCount/EventScanCount.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+			return <Loader/>;
+		}
+		return (
+			<div className={classes.root}>
+				<div className={classes.header}>
+					<Typography className={classes.pageTitle}>
+						Scan Count Report
+					</Typography>
+					<span style={{ flex: 1 }}/>
+					{eventId && !printVersion ? (
```
Comment (msbodetti) [inline src/components/pages/admin/reports/eventScanCount/EventScanCount.js]:
```suggestion
					{eventId && !printVersion && (
```

File: src/components/pages/admin/reports/eventScanCount/EventScanCount.js [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+							<Button
+								href={`/exports/reports/?type=scan_count&event_id=${eventId}`}
+								target={"_blank"}
+								iconUrl="/icons/pdf-active.svg"
+								variant="text"
+							>
+								Export PDF
+							</Button>
+						</div>
+					) : null}
```
Comment (msbodetti) [inline src/components/pages/admin/reports/eventScanCount/EventScanCount.js]:
```suggestion
					)}
```

File: src/components/pages/admin/reports/eventScanCount/EventScanCount.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+				</div>
+
+				{eventId ? (
```
Comment (msbodetti) [inline src/components/pages/admin/reports/eventScanCount/EventScanCount.js]:
```suggestion
				{eventId && (
```

File: src/components/pages/admin/reports/eventScanCount/EventScanCount.js [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+					<EventScanCountTable totals={totals} scanCounts={scanData}/>
+				) : null}
```
Comment (msbodetti) [inline src/components/pages/admin/reports/eventScanCount/EventScanCount.js]:
```suggestion
				)}
```

File: src/components/pages/admin/reports/eventScanCount/EventScanCountRow.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+					[classes.headingText]: heading,
+					[classes.subHeading]: subHeading,
+					[classes.text]: !heading,
+					[classes.totalText]: total
+				})}
+				key={index}
+				style={columnStyles[index]}
+			>
+				{text}
+				{text === "Not Scanned" ? (
```
Comment (msbodetti) [inline src/components/pages/admin/reports/eventScanCount/EventScanCountRow.js]:
```suggestion
				{text === "Not Scanned" && (
```

File: src/components/pages/admin/reports/eventScanCount/EventScanCountRow.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+					<div className={classes.noShowText}>No Show</div>
+				) : null}
```
Comment (msbodetti) [inline src/components/pages/admin/reports/eventScanCount/EventScanCountRow.js]:
```suggestion
				)}
```

**Pull request merged**